pipelines/pipelines/release/resources.yaml

```diff
@@ -0,0 +1,68 @@
+#
+# Copyright contributors to the Galasa project 
+# 
+apiVersion: tekton.dev/v1beta1
+kind: Pipeline
+metadata:
+  name: branch-resources
+  namespace: galasa-build
+spec:
+  params:
+  - name: fromBranch
+    type: string
+    default: main
+  - name: toBranch
+    type: string
+    default: main
+  - name: refspec
+    type: string
+    default: refs/heads/main:refs/heads/main
+  - name: imageTag
+    type: string
+    default: main
+  workspaces:
+  - name: git-workspace
+  tasks:
+  - name: clone-automation
+    taskRef: 
+      name: git-clone
+    params:
+    - name: url
+      value: https://github.com/galasa-dev/automation
+    - name: revision
+      value: main
+    - name: refspec
+      value: refs/heads/main:refs/heads/main
+    - name: depth
+      value: "99999999"
+    - name: subdirectory
+      value: $(context.pipelineRun.name)/automation
+    workspaces:
+     - name: output
+       workspace: git-workspace
+
+  - name: branch-docker-build-resources
+    taskRef:
+      name: docker-build
+    runAfter:
+      - clone-automation
+    params:
+    - name: pipelineRunName
+      value: $(context.pipelineRun.name)
+    - name: context
+      value: $(context.pipelineRun.name)/resources
```

Comment:
Should this be /automation?

Reply:
I think we'll need to make a directory called 'resources' inside the 'release' one. Add the httpd.conf file (just copy and paste) into the automation/dockerfiles/release/resources, and set the context for this to be that directory.